nvi-rest/src/main/java/no/sikt/nva/nvi/CandidateResponse.java

```diff
@@ -2,92 +2,35 @@
 
 import com.fasterxml.jackson.annotation.JsonAutoDetect;
 import com.fasterxml.jackson.databind.annotation.JsonSerialize;
-import java.math.BigDecimal;
 import java.net.URI;
 import java.util.ArrayList;
 import java.util.List;
-import java.util.UUID;
-import no.sikt.nva.nvi.common.db.Candidate;
-import no.sikt.nva.nvi.common.db.model.DbApprovalStatus;
-import no.sikt.nva.nvi.common.db.model.DbInstitutionPoints;
-import no.sikt.nva.nvi.common.db.model.DbNote;
 import no.sikt.nva.nvi.rest.fetch.ApprovalStatus;
 import no.sikt.nva.nvi.rest.fetch.Note;
-import no.sikt.nva.nvi.rest.upsert.NviApprovalStatus;
 
 @JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
 @JsonSerialize
-public record CandidateResponse(UUID id,
+public record CandidateResponse(URI identifier,
                                 URI publicationId,
                                 List<ApprovalStatus> approvalStatuses,
                                 List<Note> notes) {
 
-    public static CandidateResponse fromCandidate(Candidate candidate) {
```

Comment:
Not sure why you're moving this code at all to be honest. The scope of the PR is UUID -> URI.